From: Jiri Bohac <jbohac@suse.cz>
Patch-mainline: never, SLE12-SP22+ only
References: bsc#937216
Subject: bridge: remove br_netfilter warning

Since upstream commit 34666d46 (netfilter: bridge: move br_netfilter out of the
core) bridge does not pass bridged packets to netfilter unless a separate
module (br_netfilter) is loaded.

To avoid regressions in a service pack, we automatically load this module
using a softdep in /etc/modprobe.d
So this warning is misleading - remove it.

Signed-off-by: Jiri Bohac <jbohac@suse.cz>


---
 net/bridge/br.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/net/bridge/br.c
+++ b/net/bridge/br.c
@@ -227,10 +227,6 @@ static int __init br_init(void)
 	br_fdb_test_addr_hook = br_fdb_test_addr;
 #endif
 
-	pr_info("bridge: automatic filtering via arp/ip/ip6tables has been "
-		"deprecated. Update your scripts to load br_netfilter if you "
-		"need this.\n");
-
 	return 0;
 
 err_out5:
